ethcore/src/engines/authority_round/mod.rs

```diff
@@ -2387,7 +2448,7 @@ mod tests {
 		let (_spec, tap, accounts) = setup_empty_steps();
 		let engine = build_aura(|p| {
 			p.validators = Box::new(SimpleList::new(accounts.clone()));
-			p.step_duration = 4;
+			p.step_durations = [(0, 4)].to_vec().into_iter().collect();
```

Comment:
We need more tests for the actual transitions.

Reply:
I've added a test. Please let me know if you require more.